README.md

```diff
@@ -1 +1,14 @@
-# hw01
\ No newline at end of file
+# hw01
+
+### Shawn's Brief Biography:
+- **Major**: *Philosophy* & *Political Science*
+- **Year**: *Senior*
+- **Currently located**: *New Jersey*
+- Have you ever wondered if the Divorce rate in Maine correlates with the Per capita consumption of margarine? Fear not, check out this website on: [spurious relationships](https://www.tylervigen.com/spurious-correlations)
+
+### GitHub Reflections:
```

Comment:
Feel free to include your github reflections in only your pull request comment. It doesn't really matter that you've included it redundantly here but that's typically where graders will be looking for your comments.